R/utils.R

```diff
@@ -193,3 +193,19 @@ gamma_mucv2shapescale <- function(mu, cv) {
   scale <- mu * cv^2
   list(shape = shape, scale = scale)
 }
+
+
+mirror <- function(x) c(x, rev(x))
```

Comment:
```suggestion
mirror <- function(x) {
  c(x, rev(x))
}
```

Reply:
I also wonder if this should be `mirror_strain()` unless this will be useful elsewhere